nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -849,16 +845,6 @@ in
         description = "Path to DH parameters file.";
       };
 
-      proxyResolveWhileRunning = mkOption {
-        type = types.bool;
-        default = false;
-        description = ''
-          Resolves domains of proxyPass targets at runtime
-          and not only at start, you have to set
-          services.nginx.resolver, too.
```

Comment:
I think part of the reasoning here was that not resolving it at startup doesn't break nginx entirely which doesn't happen.

Reply:
good point. We should mention that here otherwise it is a bit of guesswork why we need this option exactly.